whitepapers/Tables.md

```diff
@@ -21,18 +24,18 @@ Identifier for a specific document from a specific source.
 | Column             | Type | Indexed | Nullable | FK  | Default | Description                                    |
 | :----------------- | :--: | :-----: | :------: | :-: | :-----: | :--------------------------------------------- |
 | `identifier`       | text |         |          |     |         | Identifier given to the document by the source |
-| `ingest_config_id` | int  |         |          |  ✓  |         | IngestConfig used to ingest the document       |
+| `ingest_config_id` | int  |    ✓    |          |  ✓  |         | IngestConfig used to ingest the document       |
```

Comment:
This is true, but seems redundant because FK implies an index (is why it wasn't already checked). I guess it's better to be explicit, though 👍